core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/internal/validation/ValidationBootstrap.java

```diff
@@ -76,48 +80,64 @@ public void preRun(SnowOwlConfiguration configuration, Environment env) throws E
 			int numberOfValidationThreads = Math.max(2, Runtime.getRuntime().availableProcessors() / 2);
 			env.services().registerService(ValidationThreadPool.class, new ValidationThreadPool(numberOfValidationThreads));
 
-			// synchronize rules from a default validation-rules file
-			final File validationRulesFile = env.getConfigDirectory().toPath().resolve("validation-rules.json").toFile();
-			if (validationRulesFile.exists()) {
-				LOG.info("Synchronizing validation rules from file: " + validationRulesFile);
-				final List<ValidationRule> availableRules = mapper.readValue(validationRulesFile, new TypeReference<List<ValidationRule>>() {});
+			
+			final List<File> listOfFiles = Arrays.asList(env.getConfigDirectory().listFiles());
+			final Set<File> validationRuleFiles = Sets.newHashSet();
+			final Pattern validationFilenamePattern = Pattern.compile("(validation-rules)-(\\w+).(json)");
+			for (File file : listOfFiles) {
+				final String fileName = file.getName();
+				final Matcher match = validationFilenamePattern.matcher(fileName);
+				if (match.matches()) {
+					validationRuleFiles.add(file);
+				}
+			}
+			
+			final List<ValidationRule> availableRules = Lists.newArrayList();
+			for (File validationRulesFile : validationRuleFiles) {
+				if (validationRulesFile.exists()) {
```

Comment:
Since you're collecting the files via `File.listFiles()` this check is unnecessary.